packages/base/src/StaticAreaItem.js

```diff
@@ -105,26 +53,27 @@ class StaticAreaItem {
 	 * Returns reference to the DOM element where the current fragment is added.
 	 */
 	async getDomRef() {
-		if (!this._rendered || !this.staticAreaItemDomRef) {
-			this._updateFragment();
+		this._updateContentDensity();
+		if (!this._rendered) {
+			this._rendered = true;
+			updateShadowRoot(this.ownerElement, true);
```

Comment:
the only place to render the static area content is depending on a side effect - when the control gets a dom reference.

also it is quite strange to have an async getter (with the rendering side effect).

as discussed, let's change this in a separate change where components should be responsible to wrap their static content in an `if` statement, as they best know their own state (open).